app/controllers/get_involved_controller.rb

```diff
@@ -0,0 +1,94 @@
+    end
+
+    @content_item = PresenterBuilder.new(
+      content_item,
+      content_item_path,
+      view_context,
+    ).presenter
+  end
+
+  def load_get_involved_data
+    @open_consultation_count = retrieve_given_document_type("open_consultation")['total']
+    @closed_consultation_count = retrieve_date_filtered_closed_consultations(12)
+    @next_closing_consultations = [retrieve_next_closing] 
+    @recently_opened_consultations = retrieve_new_consultations
+    @recent_consultation_outcomes = retrieve_consultation_outcomes
+    @take_part_pages = sort_take_part(retrieve_given_document_type("take_part")['results'])
+  end
+
+  def retrieve_given_document_type(document_type)
+    Services.publishing_api.get_content_items(document_type: document_type).to_hash
+  end
+
+  def retrieve_special(params)
+    Services.publishing_api.get_content_items(params).to_hash
+  end
+
+  # Aims to find a count of consultations closed in the last n months.
+  # Could feasibly be done via SearchAPI calls but there is no direct or indirect way to search by closing date
+  # there so we have to go around the houses a bit via a single large P-API call.
+  def retrieve_date_filtered_closed_consultations(months)
+    closed_count = 0
+
+    cl_docs = retrieve_special({document_type: 'closed_consultation', per_page: 500, fields: ['details']})['results']
+    cl_docs.sort_by! { |k| -k['details']['closing_date'] }.reverse!
+
+    cl_docs.each do |doc|
+      if DateTime.parse(doc['details']['closing_date']) > DateTime.now.prev_month(months)
+        closed_count += 1
+      else
+        break
+      end
+    end
+
+    return closed_count # hard return for clarity
+  end
+
+  def retrieve_next_closing
+    open_consults = retrieve_given_document_type("open_consultation")['results']
+    open_consults.sort_by! { |k| k['details']['closing_date'] }[0]
+  end
+
+  def retrieve_new_consultations
+    open_consults = retrieve_given_document_type("open_consultation")['results']
+    sorted_desc = open_consults.sort_by! { |k| k['details']['opening_date'] }.reverse!.values_at(0..2)
+    parse_organisation_acronyms(sorted_desc)
+  end
+
+  def retrieve_consultation_outcomes
+    closed_consults = retrieve_given_document_type("consultation_outcome")['results']
+    sorted_desc = closed_consults.sort_by! { |k| k['details']['closing_date'] }.reverse!.values_at(0..2)
+    parse_organisation_acronyms(sorted_desc)
+  end
+
+  def parse_organisation_acronyms(consultations)
+    consultations.each do |consultation|
+      org_acronyms = []
+      organisations = consultation['links']['organisations']
+      organisations.each do |org_id|
+        org_acronyms << Services.publishing_api.get_live_content(org_id).parsed_content['details']['acronym']
+      end
+      consultation['links']['organisation_acronyms'] = org_acronyms
+    end
+  end
+
+  def sort_take_part(take_part_pages)
+    take_part_pages.sort_by{ |page| page["details"]["ordering"] }
```

Comment:
As this only saves adding a `.sort_by` call in `load_get_involved_data`, I question the value in having it in its own function (unless you're writing a test for it, but even then there's an argument that it should be declared below a `private` keyword, thus have no 'testable' advantage to being in its own method). I think I'd put it inline instead.